docs/architecture/third-party/drupal/module-update-policy.md

```diff
@@ -0,0 +1,19 @@
+# Module update policy
+
+Drupal modules will be updated using Drupal core provided hooks, mainly:
+
+- [`hook_update_N`][1]
+- [`hook_post_update_NAME`][2]
+
+From Drupal core 8.7.x `hook_post_update_NAME` hooks are executed alphabetically and not in the order with which they
+appear in the `MODULE.post_update.php` file.
+
+For this reason we have to prepend `_NAME` with the same numeric value that we use on `hook_update_N`, meaning:
+
+- First 1 or 2 digits for Drupal core compatibility (Drupal 8, 9, 10, etc.).
+- Following 1 digit for your module's major release version; for example, `0.x` use `0`, `1.x` use `1`, for `2.x` use `2`, etc.
+- Following 2 digits for sequential counting, starting with `01`. **Note that the `x000` number can never be used**: the lowest
+update number that will be recognized and run for major version `x` is `x001`.
+
```

Comment:
Can we add the link  https://www.drupal.org/docs/8/api/update-api/introduction-to-update-api-for-drupal-8